x/nft/blockchain/handler_test.go

```diff
@@ -36,21 +40,51 @@ func TestHandleIssueTokenMsg(t *testing.T) {
 		{ // happy path
 			owner:   alice.Address(),
 			id:      []byte("other_netowork"),
-			details: blockchain.TokenDetails{[]blockchain.Network{}},
+			details: blockchain.TokenDetails{Chain: blockchain.Chain{MainTickerID: []byte("IOV")}, Iov: blockchain.IOV{Codec: "test"}},
 		},
 		{ // valid approvals
 			owner:   alice.Address(),
 			id:      []byte("other_netowork1"),
-			details: blockchain.TokenDetails{[]blockchain.Network{}},
+			details: blockchain.TokenDetails{Chain: blockchain.Chain{MainTickerID: []byte("IOV")}, Iov: blockchain.IOV{Codec: "test"}},
 			approvals: []nft.ActionApprovals{{
 				Action:    nft.Action_ActionUpdateDetails.String(),
 				Approvals: []nft.Approval{{Options: nft.ApprovalOptions{Count: nft.UnlimitedCount}, Address: bob.Address()}},
 			}},
 		},
+		{ // invalid ticker
+			owner:   alice.Address(),
+			id:      []byte("other_netowork2"),
```

Comment:
netowork?

Reply:
these are just copypasted tests, contents are not relevant, but will fix

Reply:
done